nf_core/__main__.py

```diff
@@ -269,10 +346,19 @@ def create_params_file(pipeline, revision, output, force, show_hidden):
 )
 @click.option("-o", "--outdir", type=str, help="Output directory")
 @click.option(
-    "-x", "--compress", type=click.Choice(["tar.gz", "tar.bz2", "zip", "none"]), help="Archive compression type"
+    "-x",
+    "--compress",
+    type=click.Choice(["tar.gz", "tar.bz2", "zip", "none"]),
+    help="Archive compression type",
 )
 @click.option("-f", "--force", is_flag=True, default=False, help="Overwrite existing files")
-@click.option("-t", "--tower", is_flag=True, default=False, help="Download for seqeralabs® Nextflow Tower")
+@click.option(
+    "-t",
+    "--tower",
+    is_flag=True,
+    default=False,
+    help="Download for seqeralabs® Nextflow Tower",
```

Comment:
Should probably rename this option, but until then...
```suggestion
    help="Download for Seqera Platform (formerly Nextflow Tower)",
```